src/main/java/com/model/Account.java

```diff
@@ -0,0 +1,18 @@
+package com.model;
+
+import lombok.*;
+
+@Getter
```

Comment:
You can use @Data instead @Getter @Setter

Reply:
> You can use @DaTa instead @getter @Setter

done